twint/src/main/java/com/adyen/checkout/twint/Twint.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 17/11/2023.
+ */
+
+package com.adyen.checkout.twint
+
+import androidx.activity.ComponentActivity
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import ch.twint.payment.sdk.Twint
+import ch.twint.payment.sdk.TwintPayResult
+import com.adyen.checkout.core.exception.CheckoutException
+
+object Twint {
```

Comment:
Let's rename this to avoid merchants confusing it with `ch.twint.payment.sdk.Twint`?

Reply:
No then it makes sense to keep it like this 👍